src/styles/blocks/card.scss

```diff
@@ -0,0 +1,119 @@
+.card {
+  display: flex;
+  width: 200px;
+  height: 408px;
```

Comment:
Consider removing height and allowing the card to inherit the height from its content